### PR TITLE
Wikipedia summary GUI

### LINKED ISSUE #671: Wikipedia Summary Generator (GUI)
# Aim

To display summary of a topic entered by the user 

# Details

When a user searches for a topic, a summary is displayed in the results box

### PR DESCRIPTION
**Pull request by Ayushjain2205**
# Description

Script added to generate summary about any topic from wikipedia using a GUI

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
There is some bug parsing the query from user:

image

image

**Comment by Ayushjain2205** [comment]
@kaustubhgupta  looking into it...will update you once I figure it out

**Comment by kaustubhgupta** [comment]
@Ayush7614 did you go the workaround?

**Comment by kaustubhgupta** [comment]
or shall I help in debugging?

**Comment by Ayushjain2205** [comment]
@kaustubhgupta I have made the changes , can you please test it again and let me know if it is fine now

File: Wikipedia-Summary-GUI/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+wikipedia
```
Comment (kaustubhgupta) [inline Wikipedia-Summary-GUI/requirements.txt:1]:
```suggestion
pymediawiki
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Yes, the error has been resloved. Now change the requirements.txt as suggested

**Comment by Ayushjain2205** [comment]
Done, I have updated the requirements file

**Review by kaustubhgupta — APPROVED** [review]
Cool @Ayushjain2205 💯

**Review by santushtisharma10 — APPROVED** [review]
@Ayushjain2205 💯 💯

**Comment by santushtisharma10** [comment]
@antrikshmisri

**Review by antrikshmisri — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM

**Pull request merged**